app/controllers/message_threads/bulk/signings_controller.rb

```diff
@@ -0,0 +1,48 @@
+module MessageThreads
+  module Bulk
+    class SigningsController < ::ApplicationController
+      before_action :set_message_ids
+      before_action :set_message_objects
+      before_action :nothing_to_sign_redirect, only: %i[new start]
+
+      include TurboReload
+
+      def new
+        authorize MessageObjectsTag
+      end
+
+      def start
+        authorize MessageObjectsTag
+      end
+
+      def update
+        authorize MessageObjectsTag
+
+        if params[:result] == "ok"
+          redirect_back fallback_location: message_threads_path, notice: t("signing.processed"), status: 303
+        else
+          redirect_back fallback_location: message_threads_path, alert: t("signing.failed"), status: 303
+        end
```

Comment:
Toto je strasny hack, to uz je asi lepsie tie turbo streams pouzit.

Reply:
teraz, co z toho je hack? lebo ten `status: 303` pouzivame aj inde.